src/main.py

```diff
@@ -215,6 +215,10 @@ def _generate_interaction_results(
     if CFG.make_interaction_videos:
         video = []
     for request in requests:
+        if request.train_task_idx >= CFG.max_initial_demos and \
```

Comment:
shouldn't this be `if request.train_task_idx < CFG.max_initial_demos`?

Reply:
thanks, brainfart...